File: server/src/main/java/com/spring/dishcovery/recipe/domain/entity/RecipeComment.java
package com.spring.dishcovery.recipe.domain.entity;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class RecipeComment {
    private Long commentId;
    private String recipeId;
    private String userId;
    private String content;
    private LocalDateTime createdAt;
}

File: app/Dishcovery/RecipeFeed/RecipeComment.swift
import Foundation

struct RecipeComment: Identifiable, Codable {
    var id: Int { commentId }
    let commentId: Int
    let recipeId: String
    let userId: String
    let userName: String?
    let content: String
    let createdAt: String?
}

struct RecipeLikeStatus: Codable {
    let likeCount: Int
    let liked: Bool
}
